felix/rules/static_test.go

```diff
@@ -78,6 +78,13 @@ var _ = Describe("Static", func() {
 
 	for _, trueOrFalse := range []bool{true, false} {
 		kubeIPVSEnabled := trueOrFalse
+		var dropAction Action
+		dropAction = DropAction{}
+		dropActionString := "DROP"
+		if kubeIPVSEnabled {
```

Comment:
nit: I get that this is functionally the same as using `trueOrFalse` but I think it would be a little more clear that this is not related to IPVS to have it use `trueOrFalse` instead of `kubeIPVSEnabled`